src/Microsoft.Azure.WebJobs.Extension.Storage/Blobs/Listeners/SharedBlobListenerFactory.cs

```diff
@@ -3,19 +3,18 @@
 
 using System;
 using System.Diagnostics.CodeAnalysis;
-using Microsoft.Azure.WebJobs.Host.Storage;
 using Microsoft.Azure.WebJobs.Host.Timers;
 
 namespace Microsoft.Azure.WebJobs.Host.Blobs.Listeners
 {
     internal class SharedBlobListenerFactory : IFactory<SharedBlobListener>
     {
-        private readonly IStorageAccount _account;
+        private readonly XStorageAccount _account;
         private readonly IWebJobsExceptionHandler _exceptionHandler;
         private readonly IContextSetter<IBlobWrittenWatcher> _blobWrittenWatcherSetter;
```

Comment:
Shouldn't we remove the IContextSetter interfaces completely?

Reply:
Yes. 

@MikeStall looping you as I thought you had removed these. I'll look at them if not.